essentials.yaml

```diff
@@ -1,378 +1,63 @@
-  - name: timestamper
-    version: 1.8.9
-    gav: org.jenkins-ci.plugins:timestamper:1.8.9
-    url: https://updates.jenkins.io/download/plugins/timestamper/1.8.9/timestamper.hpi
-  - name: token-macro
-    version: '2.3'
-    gav: org.jenkins-ci.plugins:token-macro:2.3
-    url: https://updates.jenkins.io/download/plugins/token-macro/2.3/token-macro.hpi
-  - name: variant
-    version: '1.1'
-    gav: org.jenkins-ci.plugins:variant:1.1
-    url: https://updates.jenkins.io/download/plugins/variant/1.1/variant.hpi
-  - name: workflow-aggregator
-    version: '2.5'
-    gav: org.jenkins-ci.plugins.workflow:workflow-aggregator:2.5
-    url: https://updates.jenkins.io/download/plugins/workflow-aggregator/2.5/workflow-aggregator.hpi
-  - name: workflow-api
-    version: '2.26'
-    gav: org.jenkins-ci.plugins.workflow:workflow-api:2.26
-    url: https://updates.jenkins.io/download/plugins/workflow-api/2.26/workflow-api.hpi
-  - name: workflow-basic-steps
-    version: '2.6'
-    gav: org.jenkins-ci.plugins.workflow:workflow-basic-steps:2.6
-    url: https://updates.jenkins.io/download/plugins/workflow-basic-steps/2.6/workflow-basic-steps.hpi
-  - name: workflow-cps
-    version: '2.45'
-    gav: org.jenkins-ci.plugins.workflow:workflow-cps:2.45
-    url: https://updates.jenkins.io/download/plugins/workflow-cps/2.45/workflow-cps.hpi
-  - name: workflow-cps-global-lib
-    version: '2.9'
-    gav: org.jenkins-ci.plugins.workflow:workflow-cps-global-lib:2.9
-    url: https://updates.jenkins.io/download/plugins/workflow-cps-global-lib/2.9/workflow-cps-global-lib.hpi
-  - name: workflow-durable-task-step
-    version: '2.19'
-    gav: org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.19
-    url: https://updates.jenkins.io/download/plugins/workflow-durable-task-step/2.19/workflow-durable-task-step.hpi
-  - name: workflow-job
-    version: '2.17'
-    gav: org.jenkins-ci.plugins.workflow:workflow-job:2.17
-    url: https://updates.jenkins.io/download/plugins/workflow-job/2.17/workflow-job.hpi
-  - name: workflow-multibranch
-    version: '2.17'
-    gav: org.jenkins-ci.plugins.workflow:workflow-multibranch:2.17
-    url: https://updates.jenkins.io/download/plugins/workflow-multibranch/2.17/workflow-multibranch.hpi
-  - name: workflow-scm-step
-    version: '2.6'
-    gav: org.jenkins-ci.plugins.workflow:workflow-scm-step:2.6
-    url: https://updates.jenkins.io/download/plugins/workflow-scm-step/2.6/workflow-scm-step.hpi
-  - name: workflow-step-api
-    version: '2.14'
-    gav: org.jenkins-ci.plugins.workflow:workflow-step-api:2.14
-    url: https://updates.jenkins.io/download/plugins/workflow-step-api/2.14/workflow-step-api.hpi
-  - name: workflow-support
-    version: '2.18'
-    gav: org.jenkins-ci.plugins.workflow:workflow-support:2.18
-    url: https://updates.jenkins.io/download/plugins/workflow-support/2.18/workflow-support.hpi
+metadata:
+  labels:
+    name: 'jenkins-essentials'
+  annotations:
```

Comment:
can/should we add a comment or so to explain what those are designed to do?

Without much context, it's hard to know what difference you're making and what its goal is for `labels` vs. `annotations`?